lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/FormDataValidator.java

```diff
@@ -52,70 +44,14 @@ ValidationResult<List<String>> validate(DrivingPermitForm drivingPermitForm) {
                 "Addresses",
                 validationErrors)) {
 
-            // Ensure addresses can be evaluated correctly
-            validationErrors.addAll(checkAddresses(drivingPermitForm.getAddresses()));
+            JsonValidationUtility.validateStringDataEmptyIsFail(
```

Comment:
Is this documented on the ticket if were updating address logic I would like to track it